examples/opencv/opencv.py

```diff
@@ -3,20 +3,32 @@
 # This is a reproduction of pachyderm's opencv example in python. A
 # walkthrough is available in the pachyderm docs:
 # https://docs.pachyderm.io/en/latest/getting_started/beginner_tutorial.html
+#
+# It makes heavy use of python_pachyderm's higher-level utility functionality
+# (`create_python_pipeline`, `put_files`), as well as more run-of-the-mill
+# functionality (`create_repo`, `create_pipeline`).
 
+import os
 import python_pachyderm
 
+def relpath(path):
+    return os.path.join(os.path.dirname(os.path.abspath(__file__)), path)
+
 def main():
     client = python_pachyderm.Client()
 
+    # Create a repo called images
     client.create_repo("images")
 
-    client.create_pipeline(
-        "edges",
-        transform=python_pachyderm.Transform(cmd=["python3", "edges.py"], image="pachyderm/opencv"),
-        input=python_pachyderm.Input(pfs=python_pachyderm.PFSInput(glob="/*", repo="images"))
+    # Create a pipeline specifically designed for executing python code. This
+    # is equivalent to the edges pipeline in the standard opencv example.
+    python_pachyderm.create_python_pipeline(
+        client,
+        relpath("edges"),
```

Comment:
So this is essentially taking whatever it finds in the `edges` directory and packaging that up and writing it into PFS? That sounds really awesome.

Reply:
Yup!